discovery_engine_core/bindings/src/types/duration.rs

```diff
@@ -88,4 +145,31 @@ mod tests {
         }
         assert_eq!(duration, *place);
     }
+
+    #[test]
+    fn test_init_option_duration() {
+        let duration = Duration::new(78978, 89378);
+        //ok as Duration: !Drop
```

Comment:
:grey_question:  what's this comment for?

Reply:
If `Duration` would be `Drop` this test would potentially leak memory as we override it without dropping it.

Reply:
I see, that's fine - just the comment was rather cryptic for me